src/tests/integration/test_foundry_prove.py

```diff
@@ -332,32 +332,20 @@ def test_foundry_dependency(
     if no_use_booster:
         pytest.skip()
 
-    dependency = 'ArithmeticContract.add(uint256,uint256)'
     test = 'ArithmeticCallTest.test_double_add(uint256,uint256)'
 
-    foundry_prove(
-        foundry,
-        tests=[(dependency, None)],
-        prove_options=ProveOptions(
-            max_iterations=10,
-            bug_report=bug_report,
-        ),
-        rpc_options=RPCOptions(
-            port=server.port,
-        ),
-    )
-
     foundry_prove(
         foundry,
         tests=[(test, None)],
         prove_options=ProveOptions(
             max_iterations=50,
             bug_report=bug_report,
+            cse=True,
         ),
         rpc_options=RPCOptions(
             port=server.port,
         ),
-        include_summaries=[(dependency, None)],
+        include_summaries=[],
```

Comment:
I guess we don't need the `include_summaries` parameter at all anymore.

Reply:
@PetarMax suggested that we should keep `include_summaries` for a while for testing purposes.